crates/bevy_pbr/src/render/mesh_bindings.rs

```diff
@@ -175,23 +174,23 @@ impl MeshLayouts {
     // ---------- BindGroup methods ----------
 
     pub fn model_only(&self, render_device: &RenderDevice, model: &BindingResource) -> BindGroup {
-        render_device.create_bind_group(&BindGroupDescriptor {
-            entries: &[entry::model(0, model.clone())],
-            layout: &self.model_only,
-            label: Some("model_only_mesh_bind_group"),
-        })
+        render_device.create_bind_group(
+            Some("model_only_mesh_bind_group"),
+            &self.model_only,
+            &[entry::model(0, model.clone())],
```

Comment:
It makes sense to not change this in this PR but the pattern used here and the rest of the file will be easy to remove in a future PR.